services/chat/chatService.ts

```diff
@@ -0,0 +1,34 @@
+'use client'
+
+export type ChatServiceSendMessageParams = {
+    anon_user_info_color_code: string;
+    anon_user_info_color_name: string;
+    anon_user_info_emoji_code: string;
+    anon_user_info_emoji_name: string;
+    member: string;
+    message: string
+}
+
+export type BetterSocialResponse<T> = {
+  success: boolean;
+  message: string;
+  data: T
+}
+
+export type SendMessageResponse = {
+  channel_id: string;
+}
+
+const sendMessage = async (data: ChatServiceSendMessageParams): Promise<BetterSocialResponse<SendMessageResponse>> => {
+  const response = await fetch('/api/chat/send', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data)
+  })
+
+  return await response.json()
+}
+
+export { sendMessage }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Please add a newline at the end of the file for better compatibility across different systems and tools.
```diff
export { sendMessage }
+ \ No newline at end of file
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->